evm/vm/forks/sharding/vm_state.py

```diff
@@ -160,14 +167,12 @@ def execute_transaction(self, transaction):
                 state_db.delta_balance(message.to, gas_refund_amount)
 
         # Miner Fees
-        transaction_fee = (transaction.gas - gas_remaining - gas_refund) * transaction.gas_price
+        transaction_fee = (transaction.gas - gas_remaining - gas_refund) * PAYGAS_gasprice
         self.logger.debug(
-            'TRANSACTION FEE: %s -> %s',
+            'TRANSACTION FEE: %s',
             transaction_fee,
-            encode_hex(self.coinbase),
         )
-        with state_db_cm() as state_db:
-            state_db.delta_balance(self.coinbase, transaction_fee)
+        computation.tx_fee = transaction_fee
```

Comment:
Rather than storing this value on the computation object what do you think about just re-calculating it within `add_transaction`?  It looks like we have all of the necessary information already available in the context of that function to do the computations.

Reply:
You are right. Totally makes sense.